server/sonar-server/src/main/java/org/sonar/server/rule/ws/SearchAction.java

```diff
@@ -182,6 +185,7 @@ protected void doDefinition(WebService.NewAction action) {
       FACET_SEVERITIES,
       FACET_ACTIVE_SEVERITIES,
       FACET_STATUSES,
+      FACET_TYPES,
```

Comment:
The description of the request parameter "facets" deserves to be completed to mention that value `types` was introduced in 5.5.